doc/source/methods/PermutationImportance.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>I think we need to make explicit the difference between <code>score_fns</code> and <code>loss_fns</code> and why there are both options available, i.e. higher-is-better vs lower-is-better and the context of the PFI score. Perhaps  it's even worth using a callout <code>Note</code> box for this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>